docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,326 @@
+
+<sup>[Download River Valley Transit example dataset](../../../assets/on-demand_services_between_multiple_zones.zip)</sup>
+
+### Define trips
+
+Similar to the previous example, because service hours vary on different days, it's necessary to define trips separately for weekdays and Saturdays. 
+
+[**trips.txt**](../../reference/#tripstxt)
+
+route_id | service_id | trip_id 
+-- | -- | -- 
+74375 | weekdays | t_5298036_b_77503_tn_0 
+74375 | saturdays | t_5298041_b_77503_tn_0 
+74375 | weekdays | t_5298046_b_77503_tn_0 
+74375 | saturdays | t_5298051_b_77503_tn_0
+
+(Define booking rules and zones using booking_rules.txt and locations.geojson in the same way as the previous example)
+
+### Define stop_times
+
+- The following data indicates that boarding is only allowed in one zone and alighting is only allowed in another zone. Boarding and alighting in the same zone are not allowed.
+
+[**stop_times.txt**](../../reference/#stop_timestxt)
+
+trip_id | location_id | stop_sequence | start_pickup_drop_off_window | end_pickup_drop_off_window | pickup_type | drop_off_type | pickup_booking_rule_id | drop_off_booking_rule_id
+-- | -- | -- | -- | -- | -- | -- | -- | --
+t_5298036_b_77503_tn_0 | area_713 | 1 | 06:30:00 | 20:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298036_b_77503_tn_0 | area_714 | 2 | 06:30:00 | 20:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298041_b_77503_tn_0 | area_713 | 1 | 09:00:00 | 19:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298041_b_77503_tn_0 | area_714 | 2 | 09:00:00 | 19:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298046_b_77503_tn_0 | area_714 | 1 | 06:30:00 | 20:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298046_b_77503_tn_0 | area_713 | 2 | 06:30:00 | 20:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+t_5298051_b_77503_tn_0 | area_714 | 1 | 09:00:00 | 19:00:00 | 2 | 1 | booking_route_74375 | booking_route_74375
+t_5298051_b_77503_tn_0 | area_713 | 2 | 09:00:00 | 19:00:00 | 1 | 2 | booking_route_74375 | booking_route_74375
+
+## On-demand services where riders must board and alight at specific locations
+
+In certain demand-responsive services, riders are unable to specify boarding and alighting at any location within a zone. Instead, riders can only book to board and alight at specific designated stops(collection points/ virtual stops). An example of this is the [RufBus service](https://uvg-online.com/rufbus-angermuende/) in Angermünde and Gartz, Germany:
+
+### Define trips
+
+Route 476 offers on-demand services between each stop in the Angermünde region, including two trips(services): one for weekdays and the other for weekends. 
```

Comment:
I am a little confused by 
- the "including": do they offer this only for two trips that run during the night, or do they just operate two trips in total?
- the "trips(services)", given that trips and services are different concepts in gtfs.

Suggestion (assuming what the above means here, feel free to update):
_Route 476 offers on-demand services between each stop in the Angermünde region. They operate two services that each have one single trip associated._

Reply:
Updated text